src/registry/FileSystemRegistry.ts

```diff
@@ -83,6 +89,17 @@ export class FileSystemRegistry extends SynchronousRegistry implements IRegistry
     this.removeFiles(Object.values(chainFiles));
   }
 
+  addWarpRoute(config: WarpCoreConfig): void {
+    let { configPath, addressesPath } = this.getWarpRoutesArtifactPaths(config);
+
+    configPath = path.join(this.uri, configPath);
+    this.createFile({ filePath: configPath, data: toYamlString(config, SCHEMA_REF) });
+
+    addressesPath = path.join(this.uri, addressesPath);
+    const addresses = warpConfigToWarpAddresses(config);
+    this.createFile({ filePath: addressesPath, data: toYamlString(addresses) });
+  }
```

Comment:
why move?

Reply:
To have this public above the protected methods. I shouldn't have put it below before